rpc-jobs/jjb-setup.yml

```diff
@@ -0,0 +1,43 @@
+- job:
+    name: 'Jenkins-Job-Builder'
+    description: Creates and updates jobs with Jenkins Job Builder.
+    logrotate:
+        daysToKeep: 20
+    node: general
+    parameters:
+        - string:
+            name: JOBS
+            description: "Which jobs to update and with what options."
+            default: -r rpc-jobs
+        - bool:
+            name: IGNORE_CACHE
+            description: "Ignore cache when updating jobs."
+        - string:
+            name: JJB_REPO
+            default: https://github.com/rcbops/rpc-gating
+        - string:
+            name: JJB_BRANCH
+            default: master
+    scm:
+        - git:
+            url: $JJB_REPO
+            branches:
+                - $JJB_BRANCH
+            skip-tag: true
+            wipe-workspace: false
+    builders:
+        - shell: |
+            scl enable python27 bash
+
+            if [[ ! -d ".venv" ]]; then
+                virtualenv -p /opt/rh/python27/root/usr/bin/python .venv
+            fi
+            source .venv/bin/activate
+
+            pip install jenkins-job-builder
+
+            if [ "$IGNORE_CACHE" = "true" ]; then
+                JJB_ARGS="--ignore-cache"
+            fi
+
+            jenkins-jobs --conf ~/jenkins_jobs.ini $JJB_ARGS update $JOBS
```

Comment:
Where does jenkins_jobs.ini come from? Could we write it out using creds from Jenkins Credential storage? Then this will work on other slaves. Or pull the config file from the repo and replace creds placeholders with creds from Jenkins.

Also this could be a template+project with an operation axis, so that we can create an update job for post merge and syntax check for PRs.